pkg/bindings/test/containers_test.go

```diff
@@ -550,21 +550,28 @@ var _ = Describe("Podman containers ", func() {
 		filtersIncorrect := map[string][]string{
 			"status": {"dummy"},
 		}
-		pruneResponse, err := containers.Prune(bt.conn, new(containers.PruneOptions).WithFilters(filtersIncorrect))
+		_, err = containers.Prune(bt.conn, new(containers.PruneOptions).WithFilters(filtersIncorrect))
+		Expect(err).ToNot(BeNil())
+
+		// List filter params should not work with prune.
+		filtersIncorrect = map[string][]string{
+			"name": {"top"},
+		}
+		_, err = containers.Prune(bt.conn, new(containers.PruneOptions).WithFilters(filtersIncorrect))
 		Expect(err).ToNot(BeNil())
 
 		// Mismatched filter params no container should be pruned.
 		filtersIncorrect = map[string][]string{
-			"name": {"r"},
+			"label": {"xyz"},
 		}
-		pruneResponse, err = containers.Prune(bt.conn, new(containers.PruneOptions).WithFilters(filtersIncorrect))
+		pruneResponse, err := containers.Prune(bt.conn, new(containers.PruneOptions).WithFilters(filtersIncorrect))
 		Expect(err).To(BeNil())
 		Expect(len(reports.PruneReportsIds(pruneResponse))).To(Equal(0))
 		Expect(len(reports.PruneReportsErrs(pruneResponse))).To(Equal(0))
 
 		// Valid filter params container should be pruned now.
 		filters := map[string][]string{
-			"name": {"top"},
+			"until": {"0s"},
```

Comment:
This change here is causing flakes?  Any ideas why? Tests locally this seems to work fine.

Reply:
I hope https://github.com/containers/podman/pull/9902 will fix the issue. I shouldn't use until here. Sorry for that.